### PR TITLE
Specify Hornet net config supplied by .env

### PR DESCRIPTION
**Pull request by trungtt1981**
Add the ability to specify Hornet net config JSON file that is supplied by the .env.
This is to avoid direct editing the file `docker\docker-compose.yml`.
So, in the .env, a new param `HORNET_CONFIG_PATH` is introduced.

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -84,6 +84,7 @@ services:
       # - "-c"
       # - "config_testnet.json"
       # - "config_alphanet.json"
+      - "-c ${HORNET_NET_CONFIG_JSON_FILE}"
```
Comment (Alexandcoats) [inline docker/docker-compose.yml:87]:
```suggestion
      - "-c ${HORNET_CONFIG_PATH}"
```

**Review by Alexandcoats — CHANGES_REQUESTED** [review]
I think we also need to add the variable to the example `.env`

EDIT: I guess there is no example file, so maybe just add it to the README section.

**Comment by trungtt1981** [comment]
> I think we also need to add the variable to the example `.env`
> 
> EDIT: I guess there is no example file, so maybe just add it to the README section.

I added a template file `.env.template` that should be OK

File: .env.template [hunk 2]
```diff
@@ -0,0 +1,7 @@
+MONGODB_USERNAME=root
+MONGODB_PASSWORD=root
+INFLUXDB_USERNAME=root
+INFLUXDB_PASSWORD=password
+JWT_PASSWORD=password
+JWT_SALT=saltines
+HORNET_CONFIG_PATH="path to hornet config JSON file"
```
Comment (Alexandcoats) [inline .env.template:7]:
```suggestion
HORNET_CONFIG_PATH=config_testnet.json
```

**Comment by Alexandcoats** [comment]
Can you add it to the README as well?

**Comment by trungtt1981** [comment]
> Can you add it to the README as well?

added

File: .env.template [hunk 3]
```diff
@@ -0,0 +1,7 @@
+MONGODB_USERNAME=root
+MONGODB_PASSWORD=root
+INFLUXDB_USERNAME=root
+INFLUXDB_PASSWORD=password
+JWT_PASSWORD=password
+JWT_SALT=saltines
+HORNET_CONFIG_PATH=config_testnet.json
```
Comment (Alexandcoats) [inline .env.template:7]:
```suggestion
HORNET_CONFIG_PATH=config_testnet.json

```

**Review by Alexandcoats — APPROVED** [review]

**Comment by trungtt1981** [comment]
> Oh, you need to sign your commits, please.

what do you mean by `sign the commits` ??

**Comment by Alexandcoats** [comment]
https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

This repo requires all commits to be signed.

**Comment by trungtt1981** [comment]
> https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits
> 
> This repo requires all commits to be signed.

Can I still sign the commits that are already pushed?
I realize that I use 2 different github accounts !!
If I use the github account that is same as the one creating the PR, then do I still need to sign.

Maybe, I should close this PR and then open another PR with only 1 github account for commit?

**Pull request closed without merging**